Bottom: db1a61a21f720b82f2c245923aa1c2bef3edca7a
Top:    e0222666bc16346a134d98b1c36d58d0e6fdc515
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-02 21:39:52 +0100

Refresh of string_l10n.patch

---

diff --git a/src/action.c b/src/action.c
index 80d666d..dde6484 100644
--- a/src/action.c
+++ b/src/action.c
@@ -7350,7 +7350,7 @@ tempfile_unlink (char * name)
     }
 
   } else {
-    fprintf (stderr, _("%s():  Unable to determine temp directory name from the temp file\n"), 
+    fprintf (stderr, _("%s():  Unable to determine temp directory name from the temp file\n"),
 	     __FUNCTION__);
     fprintf (stderr, "%s():  \"%s\"\n", 
 	     __FUNCTION__, name);
